### PR TITLE
Camel case to sentence case table heading transformer

### PR DESCRIPTION
**Pull request by vrackovic**
Transforms headings of table from camelCase to Sentence case

### CODE REVIEW

File: web/src/main/frontend/src/components/utility/camelToSentenceTransformer.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const camelToSentenceTransformer = word => {
```
Comment (iscekic) [inline web/src/main/frontend/src/components/utility/camelToSentenceTransformer.js]: Remove `Transformer` from the name.

Comment (iscekic) [reply web/src/main/frontend/src/components/utility/camelToSentenceTransformer.js]: `insertSpaceBetweenCamelCase`? 🤔

File: web/src/main/frontend/src/components/ui/TableHeading.js [hunk 2]
```diff
@@ -1,13 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import capitalize from '../utility/capitalize';
+import camelToSentenceTransformer from '../utility/camelToSentenceTransformer';
 
 export function TableHeading({tableHeadings}) {
+  const transformer = data => capitalize(data).replace('_', ' ');
```
Comment (iscekic) [inline web/src/main/frontend/src/components/ui/TableHeading.js]: Use `R.compose` here (above the component) to create the heading title.

**Comment by vrackovic** [comment]
Closing pull request until login is properly implemented

**Pull request merged**